keps/sig-storage/1979-object-storage-support/kep.yaml

```diff
@@ -18,7 +18,11 @@ reviewers:
 approvers:
   - "@saad-ali"
   - "@xing-yang"
-editor: TBD
+  - "@ehashman"
```

Comment:
Add a "prr-approvers:" and move ehashman under that:
    https://github.com/kubernetes/enhancements/blob/master/keps/NNNN-kep-template/kep.yaml#L17